js/src/alert.js

```diff
@@ -75,15 +73,8 @@ class Alert extends BaseComponent {
   _removeElement(element) {
     element.classList.remove(CLASS_NAME_SHOW)
 
-    if (!element.classList.contains(CLASS_NAME_FADE)) {
-      this._destroyElement(element)
-      return
-    }
-
-    const transitionDuration = getTransitionDurationFromElement(element)
-
-    EventHandler.one(element, 'transitionend', () => this._destroyElement(element))
-    emulateTransitionEnd(element, transitionDuration)
+    const isAnimated = element.classList.contains(CLASS_NAME_FADE)
+    super._executeCallback(() => this._destroyElement(element), element, isAnimated)
```

Comment:
Is there a reason why you call it on the base class directly vs on the extended class (`super` vs `this`)?

Reply:
not really